frontend/src/app/epics/create-namespace-bucket.js

```diff
@@ -10,12 +10,15 @@ export default function(action$, { api }) {
     return action$.pipe(
         ofType(CREATE_NAMESPACE_BUCKET),
         mergeMap(async action => {
-            const { name, readFrom, writeTo } = action.payload;
+            const { name, readFrom, writeTo, cacheTTL } = action.payload;
 
             try {
                 const namespace = {
                     read_resources: readFrom,
-                    write_resource: writeTo
+                    write_resource: writeTo,
+                    caching: {
+                        ttl: parseInt(cacheTTL)
```

Comment:
`ParseInt` is not needed in this case because the field value should already be an integer.

Reply:
Cool.